File: Perl/fb-basic.pl
#! /usr/bin/perl

for (my $i = 0; $i < 101; $i++){
    my $str = "";
    $str = $str . "fizz" if $i % 3 == 0;
    $str = $str . "buzz" if $i % 5 == 0;
    $str = $i if $str eq "";
    print "$str\n";
}

File: Python/fb-basic.py
#! /usr/bin/env python

for x in xrange(101):
    printStr = ''
    if (x % 3 == 0):
        printStr = printStr + 'fizz'
    if (x % 5 == 0):
        printStr = printStr + 'buzz'
    if (printStr == ''):
        printStr = x

    print printStr
